### PR TITLE
UI Feedback

### PR DESCRIPTION
**Pull request by chrismgeorge**
* expandable content carrots vertically aligned

### CODE REVIEW

File: packages/space-opera/editor/index.html [hunk 1]
```diff
@@ -114,7 +114,7 @@
         <me-tabbed-panel icon="home">
           <home-container></home-container>
         </me-tabbed-panel>
-        <me-tabbed-panel icon="save">
+        <me-tabbed-panel icon="import_export">
```
Comment (chrismgeorge) [inline packages/space-opera/editor/index.html:117]: this was suggested in the outside feedback, and I think it does make more sense

File: packages/space-opera/src/components/camera_settings/components/limits_base.ts [hunk 2]
```diff
@@ -134,8 +134,6 @@ export abstract class LimitsBase extends ConnectedLitElement {
     return html`
     <me-section-row label="Minimum">
       <div class="LabelRowContent">
-        <mwc-button id="clear-min-button" unelevated icon="cancel" @click="${
```
Comment (chrismgeorge) [inline packages/space-opera/src/components/camera_settings/components/limits_base.ts:137]: I don't think these are necessary for this section, and the section does look cleaner without them

File: packages/space-opera/src/components/file_modal/file_modal.ts [hunk 3]
```diff
@@ -60,14 +64,21 @@ export class FileModalElement extends LitElement {
...
-    <paper-dialog id="file-modal" modal ?opened=${this.show}>
-      <div>
-        <input type="file" class="input" id="file-input"
-          @change="${this.onFileChange}">
-        <mwc-button unelevated icon="cancel" @click=${
-        this.onCancel}>Cancel</mwc-button>
-      </div>
-    </paper-dialog>
+      <paper-dialog id="file-modal" modal ?opened=${this.show}>
+        <div class="FileModalContainer">
```
Comment (chrismgeorge) [inline packages/space-opera/src/components/file_modal/file_modal.ts:68]: Updating modal: https://screenshot.googleplex.com/6iRHM9jzkTBrEZU.png

Comment (elalish) [reply packages/space-opera/src/components/file_modal/file_modal.ts:68]: Bigger question: why do we need a dialogue here at all? The file picker already has a cancel button; can we just launch that directly from the "import GLB" button?

File: packages/space-opera/src/components/camera_settings/components/limits_base.ts (hunk 2 above)
Comment (elalish) [reply packages/space-opera/src/components/camera_settings/components/limits_base.ts:137]: The image on your other comment was very helpful; would you mind putting a few more of those in so I know what you're talking about?

Comment (elalish) [reply packages/space-opera/src/components/camera_settings/components/limits_base.ts:137]: Wait, I think I understand; so now once you set a minimum you can't unset it anymore? That seems not right...

File: packages/space-opera/src/components/home/home.ts [hunk 4]
```diff
@@ -24,44 +24,38 @@ import {homeStyles} from '../../styles.css.js';
...
-        'Import or export your GLB models as well as insert or copy &lt;model-viewer&gt; HTML snippets.'
+    body: 'Import or export GLB models as well as model-viewer HTML snippets.'
   },
   edit: {
     icon:
         'https://fonts.gstatic.com/s/i/googlematerialicons/create/v6/grey600-24dp/1x/gm_create_grey600_24dp.png',
     header: 'Edit',
-    body: 'Adjust your lighting, hotspots, or posters in finer detail.'
+    body:
+        'Adjust model-viewer\'s parameters for lighting, hotspots, and posters.'
```
Comment (elalish) [inline packages/space-opera/src/components/home/home.ts:38]: Let's remember the brackets: `<model-viewer>`.

File: packages/space-opera/src/components/materials_panel/materials_panel.ts [hunk 5]
```diff
@@ -781,7 +781,13 @@ export class MaterialPanel extends ConnectedLitElement {
 
   render() {
     if (this.materials.length === 0) {
-      return html`<div style="color: black">No materials to edit</div>`;
+      return html`
+      <me-expandable-tab tabName="Materials" .open=${true} .sticky=${true}>
+        <div slot="content">
+          <div style="color: black">No materials to edit. Load a model to edit the materials.</div>
```
Comment (elalish) [inline packages/space-opera/src/components/materials_panel/materials_panel.ts:787]: edit its materials?

File: packages/space-opera/src/components/shared/expandable_content/expandable_section.css.ts [hunk 6]
```diff
@@ -25,16 +25,12 @@ export const styles: CSSResult = css`
   max-height: 0;
   overflow: hidden;
   padding: 0 20px;
-  transition-duration: 300ms;
```
Comment (elalish) [inline packages/space-opera/src/components/shared/expandable_content/expandable_section.css.ts:28]: no more transitions?

File: packages/space-opera/src/components/shared/section_row/section_row.css.ts [hunk 7]
```diff
@@ -18,7 +18,7 @@
 
 import {css, CSSResult} from 'lit-element';
 export const styles: CSSResult = css`:host {
-  color: #dadce0;  /* GOOGLE_GREY_300 */
```
Comment (elalish) [inline packages/space-opera/src/components/shared/section_row/section_row.css.ts:21]: I notice we've switched a bunch of greys to blacks and whites. Any particular reason?

**Review by elalish — APPROVED** [review]
Code looks fine, but some design comments.

File: packages/space-opera/src/components/file_modal/file_modal.ts (hunk 3 above)
Comment (chrismgeorge) [reply packages/space-opera/src/components/file_modal/file_modal.ts:68]: this allows us to put some of the logic with respect to managing files into one place

File: packages/space-opera/src/components/camera_settings/components/limits_base.ts (hunk 2 above)
Comment (chrismgeorge) [reply packages/space-opera/src/components/camera_settings/components/limits_base.ts:137]: this removes the X next to the setting buttons: https://screenshot.googleplex.com/A6yGzfRGLdUkLHJ.png
users can still adjust the slider, so they can set a minimum and maximum just fine

File: packages/space-opera/src/components/shared/expandable_content/expandable_section.css.ts (hunk 6 above)
Comment (chrismgeorge) [reply packages/space-opera/src/components/shared/expandable_content/expandable_section.css.ts:28]: I don't think they improve the user experience, especially if this is a situation where users may be opening and closing many sections

File: packages/space-opera/src/components/shared/section_row/section_row.css.ts (hunk 7 above)
Comment (chrismgeorge) [reply packages/space-opera/src/components/shared/section_row/section_row.css.ts:21]: they're more visually appealing with the current design

**Pull request merged**